test/test_main.cpp

```diff
@@ -341,10 +342,10 @@ void RelatedGL(const Manifold& out, const std::vector<MeshGL>& originals,
       }
       const int inTri = output.faceID.empty() ? tri : output.faceID[tri];
       ASSERT_LT(inTri, inMesh.triVerts.size() / 3);
-      ivec3 inTriangle = {inMesh.triVerts[3 * inTri],
-                          inMesh.triVerts[3 * inTri + 1],
-                          inMesh.triVerts[3 * inTri + 2]};
-      inTriangle *= inMesh.numProp;
+      ivec3 inTriangle(inMesh.triVerts[3 * inTri],
```

Comment:
Apparently, the compiler is more picky when it performs implicit conversion when putting things inside the implicit constructor. Maybe the more implicit it gets, the more likely it is an issue...